<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>480</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="font">
   <font>
    <kerning>true</kerning>
   </font>
  </property>
  <property name="windowTitle">
   <string>MatriX</string>
  </property>
  <property name="windowIcon">
   <iconset resource="RESOURCE.qrc">
    <normaloff>:/new/prefix1/src/25617.png</normaloff>:/new/prefix1/src/25617.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QLabel" name="label20">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QLabel" name="label22">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label32">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label00">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label01">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label21">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLabel" name="label23">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="label02">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="label31">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="3">
     <widget class="QLabel" name="label33">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label30">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QLabel" name="label03">
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label10">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label11">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label12">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QLabel" name="label13">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>484</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Операції</string>
    </property>
    <addaction name="action_3"/>
    <addaction name="action_4"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Про програму</string>
    </property>
    <addaction name="action_7"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_2"/>
   <addaction name="separator"/>
   <addaction name="action_8"/>
   <addaction name="action_6"/>
   <addaction name="action_3"/>
   <addaction name="action_4"/>
   <addaction name="action_5"/>
   <addaction name="separator"/>
   <addaction name="action_7"/>
   <addaction name="separator"/>
   <addaction name="action"/>
  </widget>
  <action name="action">
   <property name="icon">
    <iconset resource="RESOURCE.qrc">
     <normaloff>:/new/prefix1/src/exit1.png</normaloff>:/new/prefix1/src/exit1.png</iconset>
   </property>
   <property name="text">
    <string>Вихід</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset resource="RESOURCE.qrc">
     <normaloff>:/new/prefix1/src/new.png</normaloff>:/new/prefix1/src/new.png</iconset>
   </property>
   <property name="text">
    <string>Нова матриця</string>
   </property>
  </action>
  <action name="action_3">
   <property name="icon">
    <iconset resource="RESOURCE.qrc">
     <normaloff>:/new/prefix1/src/3.png</normaloff>:/new/prefix1/src/3.png</iconset>
   </property>
   <property name="text">
    <string>Транспонувати</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset resource="RESOURCE.qrc">
     <normaloff>:/new/prefix1/src/1.png</normaloff>:/new/prefix1/src/1.png</iconset>
   </property>
   <property name="text">
    <string>Відобразити по горизонталі</string>
   </property>
  </action>
  <action name="action_5">
   <property name="icon">
    <iconset resource="RESOURCE.qrc">
     <normaloff>:/new/prefix1/src/2.png</normaloff>:/new/prefix1/src/2.png</iconset>
   </property>
   <property name="text">
    <string>Відобразити по вертикалі</string>
   </property>
  </action>
  <action name="action_6">
   <property name="icon">
    <iconset resource="RESOURCE.qrc">
     <normaloff>:/new/prefix1/src/4.png</normaloff>:/new/prefix1/src/4.png</iconset>
   </property>
   <property name="text">
    <string>Транспонувати по другій діагоналі</string>
   </property>
  </action>
  <action name="action_7">
   <property name="icon">
    <iconset resource="RESOURCE.qrc">
     <normaloff>:/new/prefix1/src/about.png</normaloff>:/new/prefix1/src/about.png</iconset>
   </property>
   <property name="text">
    <string>Інформація</string>
   </property>
  </action>
  <action name="action_8">
   <property name="icon">
    <iconset resource="RESOURCE.qrc">
     <normaloff>:/new/prefix1/src/5.png</normaloff>:/new/prefix1/src/5.png</iconset>
   </property>
   <property name="text">
    <string>Змінити на протилежний</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="RESOURCE.qrc"/>
 </resources>
 <connections/>
</ui>
